Sources/Testing/Running/Configuration.swift

```diff
@@ -203,3 +203,61 @@ public struct Configuration: Sendable {
   /// The test case filter to which test cases should be filtered when run.
   public var testCaseFilter: TestCaseFilter = { _, _ in true }
 }
+
+extension Configuration.RepetitionPolicy {
```

Comment:
`RepetitionPolicy` does not need a snapshot type. It can just be `Codable` directly as it is pure data.